recipe/meta.yaml

```
@@ -24,9 +24,11 @@ requirements:
   run:
     - python
     - python-dateutil >=2.4
-    - typing-extensions >=3.10.0.1 # [py<38]
+    - typing-extensions >=3.10.0.1
```

```suggestion
    - typing-extensions >=3.10.0.1  # [py<39]
```
see https://github.com/joke2k/faker/blob/v24.14.1/setup.py#L76